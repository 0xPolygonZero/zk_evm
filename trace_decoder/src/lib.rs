#![feature(linked_list_cursors)]
#![feature(trait_alias)]
#![feature(iter_array_chunks)]
// TODO: address these lints
#![allow(unused)]
#![allow(private_interfaces)]

mod compact;
pub mod decoding;
mod deserializers;
pub mod processed_block_trace;
pub mod trace_protocol;
pub mod types;
pub mod utils;
