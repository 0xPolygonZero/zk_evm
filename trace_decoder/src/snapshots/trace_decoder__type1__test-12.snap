---
source: trace_decoder/src/lib.rs
expression: reshaped
---
Reshape {
    state: HashedPartialTrie {
        node: Leaf {
            nibbles: Nibbles {
                count: 64,
                packed: "0x18b289936a0874cccee65712c88cdaa0a305b004d3fda2942b2b2dc54f14f611",
            },
            value: [
                248,
                72,
                128,
                132,
                59,
                154,
                202,
                0,
                160,
                118,
                140,
                60,
                158,
                125,
                67,
                147,
                163,
                110,
                49,
                152,
                218,
                97,
                29,
                218,
                136,
                94,
                162,
                155,
                155,
                15,
                4,
                79,
                239,
                163,
                7,
                247,
                133,
                60,
                76,
                209,
                220,
                160,
                166,
                136,
                91,
                55,
                49,
                112,
                45,
                166,
                46,
                142,
                74,
                143,
                88,
                74,
                196,
                106,
                127,
                104,
                34,
                244,
                226,
                186,
                80,
                251,
                169,
                2,
                246,
                123,
                21,
                136,
                210,
                59,
            ],
        },
        hash: RwLock {
            data: None,
        },
    },
    code: {},
    storage: {
        0x18b289936a0874cccee65712c88cdaa0a305b004d3fda2942b2b2dc54f14f611: HashedPartialTrie {
            node: Leaf {
                nibbles: Nibbles {
                    count: 64,
                    packed: "0x8015657e298d35290e69628be03d91f74d613caf3afdbe09138cfa415efe2f50",
                },
                value: [
                    132,
                    222,
                    173,
                    190,
                    239,
                ],
            },
            hash: RwLock {
                data: None,
            },
        },
    },
}
